### PR TITLE
feat: display hand on screen when action is available

### PR DESCRIPTION
**Pull request by jorgdebont**

### CODE REVIEW

File: src/gamelogic/Player.hpp [hunk 1]
```diff
@@ -76,12 +79,16 @@ namespace GameLogic {
...
         void _do_action();
+        Tile* _get_facing_tile();
+        void _draw_tile_usage();
         void _next_weapon();
         void _previous_weapon();
 
         vector<Weapon*> _weapons;
         int _current_weapon_index = 0;
 
+        string _action_hand_path {"res/hand.png"};
```
Comment (ThomasvLingen) [inline src/gamelogic/Player.hpp]: make this `const`

File: src/gamelogic/Player.cpp [hunk 2]
```diff
@@ -241,4 +242,26 @@ namespace GameLogic {
...
+        CoordinateDouble new_position{this->_position.x + this->_direction.x * this->_next_tile, this->_position.y + this->_direction.y * this->_next_tile};
+        return this->_level->get_tile_in_level({(int)new_position.x, (int)new_position.y});
+    }
+
+    void Player::_draw_tile_usage()
+    {
+        if (this->_get_facing_tile()->get_action_tiletriggers().size() > 0) {
+            this->_SDL_facade.draw_image(
+                this->_action_hand_path,
+                {this->_SDL_facade.get_width() / 2 - this->_SDL_facade.get_image_width(this->_action_hand_path) / 2,
```
Comment (ThomasvLingen) [inline src/gamelogic/Player.cpp]: Why do you factor in the image width for the x, but don't factor in the image height for the y?
Seems inconsistent, yo

File: src/gamelogic/Player.cpp [hunk 3]
```diff
@@ -3,11 +3,13 @@
 //
 
 #include "Player.hpp"
+#include "../engine/SDLFacade.hpp"
 
 namespace GameLogic {
 
-    Player::Player(CoordinateDouble position)
+    Player::Player(CoordinateDouble position, SDLFacade& _SDL_facade)
```
Comment (ThomasvLingen) [inline src/gamelogic/Player.cpp]: Why the _ before the SDL_facade?
It's a parameter.

File: src/gamelogic/Player.cpp (hunk 2 above)
Comment (ThomasvLingen) [inline src/gamelogic/Player.cpp]: Also, reading an image from disk every single frame(`get_image_width`), during a state where performance is critical (Muh FPS).
Disk IO takes relatively forever (especially on HDDs), so I'm not to keen on that.
Is there not a better solution?

File: src/gamelogic/Player.hpp (hunk 1 above)
Comment (jorgdebont) [inline src/gamelogic/Player.hpp]: k

File: src/gamelogic/Player.cpp (hunk 3 above)
Comment (jorgdebont) [inline src/gamelogic/Player.cpp]: k

File: src/gamelogic/Player.cpp (hunk 2 above)
Comment (jorgdebont) [inline src/gamelogic/Player.cpp]: there is but not in the scope of this project

**Pull request merged**